index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Oumaima's Portfolio</title>
+</head>
+
+<body>
+    <header>
+        <div class="logo-placeholder">
+            <a href="#" class="logo">Ouma NJ</a>
+        </div>
+        <nav class="nav-links">
+            <ul class="menu-bar">
+                <li class="hamburger-button">
+                    <img src="./images/icon-Menu.svg" alt="Menu button" class="menu-button" id="hamburger"/>
+                </li>
+            </ul>
+        </nav>
+    </header>
+    <div  class="container" id="container">
+        <section class="main">
+            <h1 class="title"> I'm Oumaima <br />Glad to see you!</h1>
+            <p class="about-me">
+                I’m a software developer! I can help you build a product , feature
+                or website Look through some of my work and experience! If you like
+                what you see and have a project you need coded, don’t hesitate to
+                contact me.
+            </p>
+            <h4 class="social-title">LET'S CONNECT</h4>
+            <ul class="social-icons">
+                <li>
+                    <a href="#"><img src="./images/social/Twitter.svg" alt="Twitter" /></a>
+                </li>
+                <li>
+                    <a href="#"><img src="./images/social/Linkedin.svg" alt="Linkedln" /></a>
+                </li>
+                <li>
+                    <a href="#"><img src="./images/social/Vector-M.svg" alt="Gmail" /></a>
+                </li>
+                <li>
+                    <a href="#"><img src="./images/social/Github.svg" alt="Github" /></a>
+                </li>
+                <li>
+                    <a href="#"> <img src="./images/social/Hand2.svg" alt="Social media"/></a>
+                </li>
+            </ul>
+        </section>  
+    </div>
```

Comment:
- [ ] Your attitude to feedback is really impressive but I think you didn't understand the previous reviewer correctly; you will agree with me that both a `section` tag and `div` tag are sectional or containing elements to maintain some block-level behavior, except that `div` has no semantic importance and as such in appropriate cases, `section` tags should always be preferred (where applicable). So, in this case, he asked that you replace your `div`  with a `section` tag, and not nest your section tag in a `div`.
- Kindly remove the `div` and wrap all your codes that are outside of the `header` tag in a `section` tag.
_I hope you get my suggestion here, because like he told you, it will improve your Search Engine Optimization count to increase traffic and aid accessibility users too._ :wink: